perception/autoware_tensorrt_bevdet/src/data.cpp

```diff
@@ -0,0 +1,84 @@
+// Copyright 2024 AutoCore, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+#include "autoware/tensorrt_bevdet/data.hpp"
+
+#include "autoware/tensorrt_bevdet/cpu_jpegdecoder.hpp"
+
+#include <stdlib.h>
+
+#include <chrono>
+#include <cstdio>
+#include <fstream>
+#include <iostream>
+
+using std::chrono::duration;
+using std::chrono::high_resolution_clock;
+
+camParams::camParams(const YAML::Node & config, int n, std::vector<std::string> & cams_name)
+: N_img(n)
+{
+  if (static_cast<size_t>(n) != cams_name.size()) {
+    RCLCPP_ERROR(
+      rclcpp::get_logger("camParams"), "Error! Need %d camera param, but given %zu camera names!",
+      n, cams_name.size());
+  }
+  ego2global_rot = fromYamlQuater(config["ego2global_rotation"]);
+  ego2global_trans = fromYamlTrans(config["ego2global_translation"]);
+
+  lidar2ego_rot = fromYamlQuater(config["lidar2ego_rotation"]);
+  lidar2ego_trans = fromYamlTrans(config["lidar2ego_translation"]);
+
+  timestamp = config["timestamp"].as<std::int64_t>();
```

Comment:
Consider using `int64_t` instead of `std::int64_t` for better consistency with the style guide's recommendation on integer types. The `std::` prefix is usually omitted for these types.